src/core/PWSprefs.h

```diff
@@ -147,7 +147,7 @@ class PWSprefs
     MainToolBarButtons, PasswordFont, TreeListSampleText, PswdSampleText,
     LastUsedKeyboard, VKeyboardFontName, VKSampleText, AltNotesEditor,
     LanguageFile, DefaultSymbols, NotesFont, NotesSampleText, AutotypeTaskDelays,
-    AddEditFont, AddEditSampleText,
+    AddEditFont, AddEditSampleText, AltNotesEditorCmdLineParms,
```

Comment:
I suspect that it would not be applicable to all editors. Where would the file name go - before "param1" or after "param2".  Would that be true for all external editors? This semantic mirrors the same additional parameters that we already provide for browser additional parameters.